### PR TITLE
SONARAZDO-431 Communicate new SonarQube names for the extension front pages

### PR DESCRIPTION
**Pull request by 7PH**
[ticket](https://sonarsource.atlassian.net/browse/SONARAZDO-431)

As we rebrand our products from SonarCloud/SonarQube to Sonar (Server, Cloud), this change should be reflected in our product documentation and what we show users. 

To be validated:
- [x] It is possible to upgrade a published extension from the old name to the new name on the Marketplace
- [x] The new endpoint input works and is retro-compatible
- [x] The new SonarQube (Server, Cloud) extension show the new names when logging informations
  - [x] SonarQube Server
  - [x] SonarQube Cloud

## Screenshots

Extension new names and logos
image

Updating extensions from old names to new one
image

Logging
image
image
image

PR Decoration
image
image

### CODE REVIEW

File: src/extensions/sonarqube/icons/SonarQubeServer.png
Comment (lucas-paulger-sonarsource) [inline src/extensions/sonarqube/icons/SonarQubeServer.png:1]: I'm sure it's just a caching problem, but when I uploaded the new extensions to my personal org, and went to run a pipeline; I saw the old images. in fact the text also looks old.
Screenshot 2024-11-11 at 17 31 19

Did you manage to see the new details? We should expect users to see these cached values for a day or so 🤷🏻

Comment (lucas-paulger-sonarsource) [reply src/extensions/sonarqube/icons/SonarQubeServer.png:1]: ignore me... I forgot to uninstall and install the new extensions

Comment (7PH) [reply src/extensions/sonarqube/icons/SonarQubeServer.png:1]: Are you using the test extension or the real one at the moment? I expect the background of the images to be either red or orange, not transparent 🤔

Comment (7PH) [reply src/extensions/sonarqube/icons/SonarQubeServer.png:1]: image
image

Comment (lucas-paulger-sonarsource) [reply src/extensions/sonarqube/icons/SonarQubeServer.png:1]: Looks good!
Screenshot 2024-11-11 at 17 59 07
Screenshot 2024-11-11 at 17 59 57

File: src/extensions/sonarqube/tasks/SonarQubePrepare/v7/task.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "id": "15B84CA1-B62F-4A2A-A403-89B77A063157",
   "name": "SonarQubePrepare",
   "friendlyName": "Prepare Analysis Configuration",
-  "description": "Prepare SonarQube analysis configuration",
+  "description": "Prepare SonarQube Server analysis configuration",
```
Comment (lucas-paulger-sonarsource) [inline src/extensions/sonarqube/tasks/SonarQubePrepare/v7/task.json:5]: looks good 
Screenshot 2024-11-11 at 18 06 09

File: src/extensions/sonarcloud/tasks/SonarCloudPrepare/v3/task.json [hunk 2]
```diff
@@ -23,11 +23,12 @@
   ],
   "inputs": [
     {
-      "name": "SonarCloud",
+      "name": "SonarQube",
+      "aliases": ["SonarQube", "SonarCloud"],
       "type": "connectedService:sonarcloud",
```
Comment (lucas-paulger-sonarsource) [inline src/extensions/sonarcloud/tasks/SonarCloudPrepare/v3/task.json:28]: might be an issue here with why the dropdown isn't populating?

File: src/extensions/sonarcloud/tasks/SonarCloudPrepare/v3/task.json [hunk 3]
```diff
@@ -168,7 +169,7 @@
   "dataSourceBindings": [
     {
       "target": "organization",
-      "endpointId": "$(SonarCloud)",
+      "endpointId": "$(SonarQube)",
```
Comment (7PH) [inline src/extensions/sonarcloud/tasks/SonarCloudPrepare/v3/task.json:172]: issue was here for the organization auto-complete

**Review by lucas-paulger-sonarsource — APPROVED** [review]
LGTM

**Pull request merged**